### ISSUE TITLE
Select drop-down to fill all rows on a column

### ISSUE THREAD

**Issue by teggsdgreat** [optikalefx/OpenJS-Grid#102]
Hi, 
Am really impressed by your work but i have some questions
1. Is it possible to have a drop down that populates all rows in a single column with the selected value
2. I have tried to run the grid but it does not load the data, i have checked my db connection but it still doesn't load the data.

Best regards

**Comment by optikalefx** [comment]
1) Drop downs are supported see Drop Downs on http://square-bracket.com/openjs
2) Check your table names, database credentials, and the ajax call for error messages.

**Comment by teggsdgreat** [comment]
Thanks for your quick response but what i want to do for example is having 100 rows of data with same value but to be determined by user eg choose country after selecting your country in the drop down the value populates into all remaining 99 rows of the same column in the grid. Is this possible if yes, how.

Thanks again

**Comment by optikalefx** [comment]
Yes that's possible, but not built in. You'd have just traverse the data object to change those values (and then re-render), or use jQuery to change the select boxes. Neither unfortunately is going to be super easy.
